Nos/Models/NoteParser.swift

```diff
@@ -114,7 +115,7 @@ struct NoteParser {
     /// Replaces Nostr entities embedded in the note (without a proper tag) with markdown links
     private func replaceNostrEntities(in content: String) -> String {
         // swiftlint:disable opening_brace operator_usage_whitespace closure_spacing comma superfluous_disable_command
-        let unformattedRegex = /(?:^|\s)@?(?:nostr:)?(?<entity>((npub1|note1|nprofile1|nevent1)[a-zA-Z0-9]{58,255}))/
+        let unformattedRegex = /(?:^|\s)@?(?:nostr:)?(?<entity>((npub1|note1|nprofile1|nevent1)[a-zA-Z0-9]{58,}))/ // TODO: where'd we get the 58? is that right?
```

Comment:
@martindsq @mplorentz Do either of you know why we have 58 here? I haven't found anything in the NIP-19 documentation that says, but maybe this is from some other source.

Reply:
It is the length of an `npub10elfcs4fr0l0r8af98jlmgdh9c8tcxjvz9qkw038js35mp4dma8qzvjptg` after you get rid of `npub1` which is already captured.

Reply:
Is that the shortest possible length? This also looks for note, nprofile, and nevent, so it's not just npub, and I'd like to be sure we get this right. The upper limit of 255 was wrong and causing the bug that's fixed in this PR. Is it possible that npub or any of the others could be shorter?

Reply:
That is indeed the shortest possible length 👍

Reply:
Awesome -- thanks!